scripts/azureml-assets/azureml/assets/config.py

```diff
@@ -141,6 +142,76 @@ def release_paths(self) -> List[Path]:
         return release_paths
 
 
+class ModelConfig(Config):
+    """
+
+    Example:
+
+    name: bert-base-uncased
+    path: # should contain local_path or should contain package object
+        local_path: "../models/bert-base-uncased" # the local path to the model
+        package:
+            url: https://huggingface.co/bert-base-uncased
+            commit_hash: 5546055f03398095e385d7dc625e636cc8910bf2
+    publish:
+        type: mlflow_model # could be one of (custom_model, mlflow_model, triton_model)
+        flavor: hftransformers # flavor should be specificed only for mlflow_model
+        tags: # tags published to the registry
+            isv : huggingface
+            task: fill-mask
+    """
+    def __init__(self, file_name: Path):
+        super().__init__(file_name)
+        self._validate()
+
+    def _validate(self):
+        Config._validate_exists('model.name', self.name)
+
+    @property
+    def name(self) ->str: 
+        return self._yaml.get("name")
+
+    @property
+    def _path(self) -> object:
+        return self._yaml.get("path")
+
+    @property
+    def path_local_path(self) -> object:
+        return self._path.get('local_path')
+
+    @property
+    def _path_package(self) -> Dict[str,str]:
+        return self._path.get("package") if type(self._path()!="str") else None

```

Comment:
>)!=" [](http://example.com/codeflow?start=60&length=4)

Could you please run black on the file?

Reply:
Note that [Black](https://pypi.org/project/black/) isn't a requirement for the repo, but [flake8](https://flake8.pycqa.org/en/latest/) is used in one of the checks to ensure proper formatting.